orttraining/orttraining/test/python/orttraining_run_bert_pretrain.py

```diff
@@ -317,6 +335,14 @@ def setup_training(args):
     logger.info("setup_training: args.train_batch_size = %d", args.train_batch_size)
     return device, args
 
+def setup_torch_distributed(world_rank, world_size):
+    os.environ['RANK'] = str(world_rank)
+    os.environ['WORLD_SIZE'] = str(world_size)
```

Comment:
Why do we need rank and size as env var? `ORTTrainer` reads these information from `ORTTrainerOptions.distributed` sub-dict

Reply:
need to set it to use torch distributed barrier